packages/backend/src/payment-method/ilp/peer/service.ts

```diff
@@ -134,6 +142,11 @@ async function createPeer(
     return PeerError.InvalidHTTPEndpoint
   }
 
+  const asset = await deps.assetService.get(options.assetId)
```

Comment:
```suggestion
  const asset = await deps.assetService.get(options.assetId, options.tenantId)
```
so we know that the peer gets created for the correct tenant